lib/publiccloud/instance.pm

```diff
@@ -250,6 +251,7 @@ sub wait_for_ssh
     }
 
     # Debug output: We have ocasional error in 'journalctl -b' - see poo#96464 - this will be removed soon.
```

Comment:
```suggestion
    # Debug output: We have occasional error in 'journalctl -b' - see poo#96464 - this will be removed soon.
```

I know it's outside of this PR but this typo kinda bugs me ...